CHANGELOG.adoc

```diff
@@ -478,6 +478,7 @@ To illustrate this new feature, we contribute a new tool on the _Papaya Diagram_
 - https://github.com/eclipse-sirius/sirius-web/issues/2371[#2371] [view] Add new default color palettes along with the creation of a view (theme colors, mui colors...).
 These color palettes are accessible in studio projects but are not visible in the Explorer view.
 - https://github.com/eclipse-sirius/sirius-web/issues/2371[#2371] [view] Add new validation rules about colors for View diagrams & forms.
+- https://github.com/eclipse-sirius/sirius-web/issues/2789[#2789] [diagram] Increase the size of the edgeupdater to make reconnecting an edge easier.
```

Comment:
Since you add a file in sirius-web, this should be in the breaking changes section